tooling/tool-install-guide.md

```diff
@@ -0,0 +1,263 @@
+# Tool Install Guide
+This guide will walk you through installing common engineering tools on your system. If you haven't read the [computer setup](./computer-setup.md) guide yet, start there. This guide is written for Ubuntu derivatives using the BASH shell.
+
+1. Update your system.
+    ```bash
+    sudo apt-get update
+    sudo apt-get upgrade -y
+    sudo apt-get autoremove
+    ```
+1. [Terminator](https://gnome-terminator.org) - iTerm2-like terminal emulator for Linux
+    ```bash
+    sudo apt-get install -y terminator
+    ```
+    1. Check `Terminator` > right-click > `Preferences` > `Profiles` > `Scrolling` > `Infinite Scrollback`.
+    1. I like to invert the titlebar colors so "focused" is blue (`#0076C9`) or green (`#3FBC6C`), and "receiving" is red (`#C80003`). It is just less aggressive and more plesant to look at. These settings are in `Terminator` > right-click > `Preferences` > `General` > `Terminal Titlebar`.
+1. [Vim](https://www.vim.org) - CLI text editor and related utilities
+    ```bash
+    sudo apt-get install -y vim vim-common
+    ```
+1. [VScode](https://code.visualstudio.com) - GUI text editor
+    ```bash
+    sudo apt-get install -y vim vim-common
```

Comment:
This command is not installing VSCode. Also, maybe it would be worth later linking to a separate document recommending particular useful extensions for VSCode.

Reply:
Fixed, thank you.